In [1]:
import pandas as pd

In [23]:
TCGAHealthy = pd.read_csv('../Data/pradrsemfpkmtcga.txt',delimiter='\t')

In [33]:
TCGAHealthy

Hugo_Symbol  Entrez_Gene_Id  TCGA-EJ-7789-11A-01R-2118-07  \
0              CUL5            8065                        307.69   
1              REV1           51455                        503.95   
2            ZNF428          126299                       1008.90   
3            CX3CR1            1524                         85.82   
4              FAXC           84553                        239.52   
5              PKP2            5318                        239.52   
6             BNIP2             663                        674.59   
7             HOXB7            3217                        107.38   
8            GRIN3A          116443                         28.86   
9             FCRL5           83416                          9.70   
10           GPR150          285601                         15.45   
11          ADORA2A             135                         52.45   
12             YBX2           51087                         14.35   
13             CD1B             910                          2.89   
14            MORC2           22880                        368.65   
15             RHBG           57127                          4.28   
16             GART            2618                        669.92   
17            CLCN2            1181                         62.56   
18            IFIH1           64135                        320.80   
19           CTAGE8       100142659                         23.59   
20       AL035252.1               0                       1096.50   
21            SMYD3           64754                        348.71   
22             NARF           26502                       1313.23   
23         FAM177A1          283635                       3372.43   
24           PLGLB1            5342                         26.67   
25     CTC-435M10.3               0                          0.00   
26     RP11-133K1.2               0                          0.00   
27         DEFB104B          140596                          0.00   
28       AC079354.1               0                          0.35   
29            C4BPA             722                          4.28   
...             ...             ...                           ...   
19016         SIRPA          140885                        332.14   
19017          NOS1            4842                         12.64   
19018         CYTH2            9266                       1407.55   
19019        ZNF599          148103                        301.33   
19020         NOP10           55505                       5112.16   
19021        CMKLR1            1240                         68.55   
19022        ZNF812          729648                        143.01   
19023        LANCL2           55915                        341.51   
19024         PEX13            5194                        890.44   
19025         CLIP2            7461                        162.14   
19026       PPP1R1B           84152                       2468.49   
19027        EFEMP1            2202                       2133.97   
19028         NOC3L           64318                        226.54   
19029       B3GALT1            8708                          3.92   
19030         DRAM1           55332                        267.73   
19031          KLK5           25818                          3.32   
19032           HYI           81888                       1340.84   
19033       FAM215A               0                          2.34   
19034         FGF14            2259                         13.62   
19035         REEP6           92840                       1674.06   
19036       TMEM38B           55151                        329.84   
19037         GATA6            2627                         47.17   
19038       COL28A1          340267                         38.12   
19039          PDK3            5165                        136.19   
19040          MC1R            4157                         44.89   
19041         IFT74           80173                        255.00   
19042 

In [25]:
TCGACancer = pd.read_csv('../Data/pradrsemfpkmtcgat.txt',delimiter='\t')

In [28]:
TCGACancer

Hugo_Symbol  Entrez_Gene_Id  TCGA-VN-A88P-01A-11R-A352-07  \
0              CUL5            8065                        218.79   
1              REV1           51455                        420.68   
2            ZNF428          126299                       1135.20   
3            CX3CR1            1524                          5.36   
4              FAXC           84553                        100.13   
5              PKP2            5318                        214.27   
6             BNIP2             663                        688.78   
7             HOXB7            3217                         40.93   
8            GRIN3A          116443                          5.59   
9             FCRL5           83416                          2.34   
10           GPR150          285601                          2.53   
11          ADORA2A             135                         67.59   
12             YBX2           51087                          4.35   
13             CD1B             910                          1.50   
14            MORC2           22880                        688.78   
15             RHBG           57127                          5.96   
16             GART            2618                        780.44   
17            CLCN2            1181                        135.24   
18            IFIH1           64135                        190.34   
19           CTAGE8       100142659                         62.56   
20       AL035252.1               0                       3124.78   
21            SMYD3           64754                        642.59   
22             NARF           26502                       1923.14   
23         FAM177A1          283635                       2468.49   
24           PLGLB1            5342                         31.22   
25     CTC-435M10.3               0                          0.00   
26     RP11-133K1.2               0                          0.00   
27         DEFB104B          140596                          0.00   
28       AC079354.1               0                          0.36   
29            C4BPA             722                          0.79   
...             ...             ...                           ...   
19016         SIRPA          140885                        129.69   
19017          NOS1            4842                          0.61   
19018         CYTH2            9266                        802.41   
19019        ZNF599          148103                        529.06   
19020         NOP10           55505                       5329.30   
19021        CMKLR1            1240                         62.12   
19022        ZNF812          729648                         16.03   
19023        LANCL2           55915                        392.44   
19024         PEX13            5194                        376.41   
19025         CLIP2            7461                         85.82   
19026       PPP1R1B           84152                       5479.15   
19027        EFEMP1            2202                        703.28   
19028         NOC3L           64318                        214.27   
19029       B3GALT1            8708                          2.25   
19030         DRAM1           55332                         95.34   
19031          KLK5           25818                          0.00   
19032           HYI           81888                        967.76   
19033       FAM215A               0                          1.20   
19034         FGF14            2259                         24.99   
19035         REEP6           92840                       1832.01   
19036       TMEM38B           55151                        193.01   
19037         GATA6            2627                        119.26   
19038       COL28A1          340267                        215.77   
19039          PDK3            5165                         51.35   
19040          MC1R            4157                         60.82   
19041         IFT74           80173                        279.14   
19042 

In [29]:
GTEX = pd.read_csv('../Data/prostatersemfpkmgtex.txt',delimiter='\t')

In [30]:
GTEX

Hugo_Symbol  Entrez_Gene_Id  GTEX-111CU-1526-SM-5N9FS  \
0              CUL5            8065                    291.04   
1              REV1           51455                    429.54   
2            ZNF428          126299                    995.00   
3            CX3CR1            1524                     43.94   
4              FAXC           84553                    320.80   
5              PKP2            5318                    371.22   
6             BNIP2             663                    780.44   
7             HOXB7            3217                    104.42   
8            GRIN3A          116443                      6.94   
9             FCRL5           83416                      1.25   
10           GPR150          285601                      4.94   
11          ADORA2A             135                     62.56   
12             YBX2           51087                     18.16   
13             CD1B             910                      1.95   
14            MORC2           22880                    525.39   
15             RHBG           57127                      6.73   
16             GART            2618                    587.13   
17            CLCN2            1181                     73.54   
18            IFIH1           64135                    166.73   
19           CTAGE8       100142659                     35.50   
20       AL035252.1               0                    954.43   
21            SMYD3           64754                    379.04   
22             NARF           26502                   1295.13   
23         FAM177A1          283635                   2148.82   
24           PLGLB1            5342                     45.21   
25     CTC-435M10.3               0                      0.00   
26     RP11-133K1.2               0                      0.00   
27         DEFB104B          140596                      0.00   
28       AC079354.1               0                      0.00   
29            C4BPA             722                      5.36   
...             ...             ...                       ...   
19016         SIRPA          140885                    473.41   
19017          NOS1            4842                     33.54   
19018         CYTH2            9266                   1143.10   
19019        ZNF599          148103                    237.86   
19020         NOP10           55505                   4152.18   
19021        CMKLR1            1240                     77.25   
19022        ZNF812          729648                     23.08   
19023        LANCL2           55915                    283.05   
19024         PEX13            5194                    279.14   
19025         CLIP2            7461                    273.37   
19026       PPP1R1B           84152                   4972.34   
19027        EFEMP1            2202                   1733.13   
19028         NOC3L           64318                    208.38   
19029       B3GALT1            8708                      4.39   
19030         DRAM1           55332                    281.09   
19031          KLK5           25818                      0.00   
19032           HYI           81888                   1313.23   
19033       FAM215A               0                      2.32   
19034         FGF14            2259                      2.34   
19035         REEP6           92840                   1286.18   
19036       TMEM38B           55151                    165.57   
19037         GATA6            2627                    155.50   
19038       COL28A1          340267                      8.00   
19039          PDK3            5165                     94.67   
19040          MC1R            4157                     19.68   
19041         IFT74           80173                    258.57   
19042         CDK14            5218                    287.01   
19043  RP11-762I7.5               0                      4.94   
19044           GSR            2936                   1832.01   
19045        UBQLN4           56893        

In [35]:
BRCAHealthy = pd.read_csv('../Data/brcarsemfpkmtcga.txt',delimiter='\t')
BRCACancer = pd.read_csv('../Data/brcarsemfpkmtcgat.txt',delimiter='\t')
GTEXBreast = pd.read_csv('../Data/breastrsemfpkmgtex.txt',delimiter='\t')

In [36]:
BRCAHealthy

Hugo_Symbol  Entrez_Gene_Id  TCGA-BH-A1EU-11A-23R-A137-07  \
0              KIF1C           10749                       1674.06   
1             SWAP70           23075                       1088.92   
2               PAK1            5058                        748.61   
3           KRTAP9-1          728318                          0.00   
4            PLEKHM3          389072                        110.43   
5              MCFD2           90411                       1697.45   
6              RBM12           10137                        836.53   
7              KRT86            3892                         28.24   
8             PSMD12            5718                        785.88   
9            SLC7A11           23657                         10.96   
10            BTN3A1           11119                        457.25   
11             SFTPB            6439                          0.46   
12             LPIN1           23175                        723.08   
13            NT5C1A           84618                          1.10   
14           L3MBTL2           83746                        417.77   
15            SLC9A9          285195                        180.02   
16              ST7L           54879                        525.39   
17              SYT2          127833                          3.47   
18          CACNA2D4           93589                         43.02   
19             ZNF84            7637                        540.19   
20              RBP2            5948                          4.03   
21          HLA-DQB1            3119                        204.07   
22              DPP4            1803                        186.40   
23            LRRC36           55282                         15.56   
24             ATP5D             513                        780.44   
25             LAMB3            3914                       1832.01   
26            MRPL34           64981                        995.00   
27          C19orf57           79173                         64.80   
28            BNIP3L             665                       2468.49   
29            NT5DC2           64943                        571.05   
...              ...             ...                           ...   
19708           FJX1           24147                        193.01   
19709           FGF7            2252                       1167.14   
19710          TPPP2          122664                         12.09   
19711          RBM10            8241                        441.64   
19712          OPCML            4978                          2.92   
19713           CA10           56934                          0.37   
19714          PNPT1           87178                        307.69   
19715        FILIP1L           11259                        591.22   
19716           COA3           28958                       1721.16   
19717          OR1B1          347169                          0.00   
19718        SLC22A7           10864                          0.00   
19719       ARHGEF15           22899                        395.18   
19720          PLCB1           23236                        384.34   
19721          STAC3          246329                         39.79   
19722         AFG3L2           10939                        842.36   
19723        FAM120A           23196                       1628.26   
19724         SRGAP3            9901                         99.43   
19725          PRMT5           10419                       1498.22   
19726          CREG2          200407                          1.85   
19727        SLC39A7            7922                       1286.18   
19728        CCDC138          165055                         67.59   
19729        SLC39A8           64116                        575.03   
19730         ZNF221            7638                         59.13   
19731  RP11-552I14.1               0                          0.00   
19732           NNAT            4826                        126.12   
19733           EID

In [37]:
BRCACancer

Hugo_Symbol  Entrez_Gene_Id  TCGA-AC-A2BM-01A-11R-A21T-07  \
0              KIF1C           10749                        974.50   
1             SWAP70           23075                        387.02   
2               PAK1            5058                        748.61   
3           KRTAP9-1          728318                          0.00   
4            PLEKHM3          389072                         31.22   
5              MCFD2           90411                        928.30   
6              RBM12           10137                       1001.93   
7              KRT86            3892                          1.01   
8             PSMD12            5718                        974.50   
9            SLC7A11           23657                         53.19   
10            BTN3A1           11119                        320.80   
11             SFTPB            6439                          1.03   
12             LPIN1           23175                        251.48   
13            NT5C1A           84618                          0.58   
14           L3MBTL2           83746                        583.07   
15            SLC9A9          285195                         58.30   
16              ST7L           54879                        454.09   
17              SYT2          127833                          0.34   
18          CACNA2D4           93589                         55.10   
19             ZNF84            7637                        642.59   
20              RBP2            5948                         54.72   
21          HLA-DQB1            3119                       3443.31   
22              DPP4            1803                         72.52   
23            LRRC36           55282                         13.12   
24             ATP5D             513                       1639.59   
25             LAMB3            3914                        104.42   
26            MRPL34           64981                       1781.89   
27          C19orf57           79173                        103.69   
28            BNIP3L             665                       1594.73   
29            NT5DC2           64943                        356.05   
...              ...             ...                           ...   
19708           FJX1           24147                        214.27   
19709           FGF7            2252                         51.71   
19710          TPPP2          122664                          0.00   
19711          RBM10            8241                        587.13   
19712          OPCML            4978                          1.46   
19713           CA10           56934                          0.00   
19714          PNPT1           87178                        406.31   
19715        FILIP1L           11259                        314.17   
19716           COA3           28958                       2935.74   
19717          OR1B1          347169                          0.00   
19718        SLC22A7           10864                          0.00   
19719       ARHGEF15           22899                         68.07   
19720          PLCB1           23236                         18.03   
19721          STAC3          246329                         74.58   
19722         AFG3L2           10939                       1104.13   
19723        FAM120A           23196                       2018.80   
19724         SRGAP3            9901                         90.77   
19725          PRMT5           10419                        684.02   
19726          CREG2          200407                          2.10   
19727        SLC39A7            7922                       2287.20   
19728        CCDC138          165055                         94.67   
19729        SLC39A8           64116                        403.50   
19730         ZNF221            7638                         12.09   
19731  RP11-552I14.1               0                          0.00   
19732           NNAT            4826                         17.51   
19733           EID

In [38]:
GTEXBreast

Hugo_Symbol  Entrez_Gene_Id  GTEX-WYVS-1726-SM-3NMAY  \
0              KIF1C           10749                  1199.98   
1             SWAP70           23075                   836.53   
2               PAK1            5058                  1407.55   
3           KRTAP9-1          728318                     0.00   
4            PLEKHM3          389072                    78.89   
5              MCFD2           90411                  1286.18   
6              RBM12           10137                   890.44   
7              KRT86            3892                    31.00   
8             PSMD12            5718                   802.41   
9            SLC7A11           23657                    36.27   
10            BTN3A1           11119                  1794.29   
11             SFTPB            6439                     6.26   
12             LPIN1           23175                   389.72   
13            NT5C1A           84618                     0.00   
14           L3MBTL2           83746                   511.00   
15            SLC9A9          285195                   141.02   
16              ST7L           54879                   551.56   
17              SYT2          127833                     3.44   
18          CACNA2D4           93589                    44.57   
19             ZNF84            7637                   830.75   
20              RBP2            5948                     6.21   
21          HLA-DQB1            3119                  3303.00   
22              DPP4            1803                    59.55   
23            LRRC36           55282                    14.67   
24             ATP5D             513                  1001.93   
25             LAMB3            3914                  4481.23   
26            MRPL34           64981                   802.41   
27          C19orf57           79173                   171.45   
28            BNIP3L             665                  1467.37   
29            NT5DC2           64943                   656.11   
...              ...             ...                      ...   
19708           FJX1           24147                    63.89   
19709           FGF7            2252                  1259.69   
19710          TPPP2          122664                    14.03   
19711          RBM10            8241                   607.87   
19712          OPCML            4978                     0.83   
19713           CA10           56934                     1.01   
19714          PNPT1           87178                   312.00   
19715        FILIP1L           11259                  1059.11   
19716           COA3           28958                  1662.49   
19717          OR1B1          347169                     0.83   
19718        SLC22A7           10864                     0.00   
19719       ARHGEF15           22899                   237.86   
19720          PLCB1           23236                   373.81   
19721          STAC3          246329                   111.99   
19722         AFG3L2           10939                   743.43   
19723        FAM120A           23196                  1191.69   
19724         SRGAP3            9901                   167.90   
19725          PRMT5           10419                  1487.87   
19726          CREG2          200407                     2.56   
19727        SLC39A7            7922                  1322.37   
19728        CCDC138          165055                   130.60   
19729        SLC39A8           64116                   460.44   
19730         ZNF221            7638                    45.21   
19731  RP11-552I14.1               0                     0.00   
19732           NNAT            4826                    73.03   
19733           EID3          493861                   101.54   
19734         ZNF740          283337                   607.87   
19735           ELP4           26610                   323.03   
19736          SPSB2           84727                   126.12   
19737        BCL2L12           83596        

Prostate has ~3:1 (426:154) Cancerous:Healthy (48 TCGAH, 106 GTEx, 426 TCGAC)

Breast has ~5:1 (982:199) Cancerous:Healthy (110 TCGAH, 89 GTEx, 982 TCGAC)